packages/web3torrent/src/utils/torrent-status-checker.ts

```diff
@@ -42,6 +42,12 @@ export const getFormattedETA = (torrent: Torrent) => {
   ].join('');
 };
 
+export const getPeerStatus = (torrent, wire) => {
+  if (!wire) return false;
```

Comment:
I do a double check here, if the wire got dropped, or the torrent considers that the peer is no longer online, the peer is considered offline